src/main/java/com/epam/aidial/core/storage/ResourceDescription.java

```diff
@@ -19,7 +19,7 @@
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 public class ResourceDescription {
 
-    private static final Set<Character> INVALID_FILE_NAME_CHARS = Set.of('/', '{', '}');
+    private static final Set<Character> INVALID_FILE_NAME_CHARS = Set.of('/', '{', '}', '"');
```

Comment:
Is that a part of the patch?
should we add this fix to the develop branch as well ?

Reply:
this fix was requested for this specific version